File: simple_samples/fizzbuzz.go
package main

import "fmt"

// Fizz if divisible by 3, buzz by 5, fizzbuzz for both
func main() {
	fmt.Println("~Fizzbuzz~")
	printFizzBuzzRange(0, 25)
}

func printFizzBuzzRange(start, end int) {
	for ; start <= end; start++ {
		printFizzBuzz(start)
	}
}

func printFizzBuzz(specific int) {
	if specific%3 == 0 && specific%5 == 0 {
		fmt.Println("FizzBuzz")
	} else if specific%3 == 0 {
		fmt.Println("Fizz")
	} else if specific%5 == 0 {
		fmt.Println("Buzz")
	} else {
		fmt.Println(specific)
	}
}

File: simple_samples/fizzbuzz.cpp
#include <iostream>

void printFizzBuzz(int, int);
void printFizzBuzz(int);

int main() {
    std::cout << "~FizzBuzz~" << std::endl;
    printFizzBuzz(0,25);
}

void printFizzBuzz(int start, int end) {
    for (; start<=end; start++) {
        printFizzBuzz(start);
    }
}

void printFizzBuzz(int specific) {
    if (specific % 3 == 0 && specific % 5 == 0) {
        std::cout << "FizzBuzz" << std::endl;
    } else if (specific % 3 == 0) {
        std::cout << "Fizz" << std::endl;
    } else if (specific % 5 == 0) {
        std::cout << "Buzz" << std::endl;
    } else {
        std::cout << specific << std::endl;
    }
}